articles/virtual-machines/linux/multiple-nics.md

```diff
@@ -176,4 +177,75 @@ You can also use a `copyIndex()` to then append a number to a resource name, whi
 You can read a complete example of [creating multiple NICs using Resource Manager templates](../../virtual-network/virtual-network-deploy-multinic-arm-template.md).
 
 ## Next steps
-Review [Linux VM sizes](sizes.md) when trying to creating a VM with multiple NICs. Pay attention to the maximum number of NICs each VM size supports. 
\ No newline at end of file
+Review [Linux VM sizes](sizes.md) when trying to creating a VM with multiple NICs. Pay attention to the maximum number of NICs each VM size supports. 
+
+## Additional Steps After A Linux VM with two NICs Got Created

```

Comment:
This section should move up above 'Next steps'. The H2 should be renamed to something like 

> Configure guest OS for multiple NICs